### PR TITLE
Update Symfony to 4.4.40

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update Symfony to 4.4.40 . This will protect `develop` against [CVE-2021-41270](https://nvd.nist.gov/vuln/detail/CVE-2021-41270)
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| Related PRs       | 
| How to test?      | Regressions tests
| Possible impacts? | No expected impact as this is a symfony patch version upgrade

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/28273)

### CODE REVIEW

**Review by kpodemski — DISMISSED** [review]

File: composer.lock [hunk 1]
```diff
@@ -12743,5 +12743,5 @@
     "platform-overrides": {
         "php": "7.2.5"
     },
-    "plugin-api-version": "2.2.0"
+    "plugin-api-version": "2.1.0"
```
Comment (Progi1984) [inline composer.lock]: Could you revert this change ?

**Review by Progi1984 — COMMENTED** [review]
A small comment and it's ok

File: composer.lock (hunk 1 above)
Comment (Seb33300) [reply composer.lock]: You need to do `composer self-update`

Comment (matks) [reply composer.lock]: Done

**Review by Progi1984 — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Review by HanaRebaiQA — APPROVED** [review]
Hello @matks 

Automated tests are OK

So, it's QA :heavy_check_mark: 

Thanks!

**Comment by Progi1984** [comment]
Thanks @HanaRebaiQA & @matks

**Pull request merged**